test/wpt/status/streams.json

```diff
@@ -23,26 +17,13 @@
     "skip": "Browser-specific test"
   },
   "readable-streams/owning-type-message-port.any.js": {
-    "fail": {
-      "expected": [
-        "Transferred MessageChannel works as expected",
-        "Second branch of owning ReadableStream tee should end up into errors with transfer only values"
-      ]
-    }
+    "skip": "Readable streams with type owning are not yet supported"
```

Comment:
I think we should keep the expected failures and not skip so long as the test file runs and doesn't get stuck or similar.

Reply:
Fair point. I'll change it back to an expected failure and add a note instead.